app/models/ability.rb

```diff
@@ -0,0 +1,37 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    can :read, Post
+    return unless user.present?
+
+    can :manage, Post, author_id: user.id
+    can :manage, Comment, author_id: user.id
+
+    return unless user.role == 'admin'
+
+    can :manage, Post
+    can :manage, Comment
```

Comment:
![image](https://github.com/Lul-Abdifan/Blog-App/assets/93039359/cbbeef72-8b96-4f28-8d0f-b8fa95ff9d86)

- [ ] Great work so far 👍 but it would be great if you add a checkbox or input field to add if the user is an admin or not because right now I cannot create a user with admin role 😨 Kindly fix this issue so next reviewer can check if your app behave in an expected way for the user with role admin 😊